Ginger/GingerCoreCommon/GeneralLib/ObservableList.cs

```diff
@@ -335,37 +329,19 @@ public void LoadLazyInfo()
             {
                 loadingata = true;
                 // We need 2nd check as it might changed after lock released
+                if (!LazyLoad) return;   //since several functions can try to get data we need to lock and verify before we convert 
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)